package.json

```diff
@@ -38,13 +38,15 @@
     "build:watch": "tsc -w",
     "prebuild": "yarn run clean",
     "build": "yarn run copy-templates && tsc --pretty",
-    "prepare": "node scripts/husky-install && cd frontend && yarn && cd .. && yarn run build",
+    "build:frontend": "yarn --cwd ./frontend",
```

Comment:
Maybe this one could explicitly build `frontend` instead of relying on frontend's `prepare` script? That way maybe we could also get rid of `prepare` script in frontend and be more explicit instead.

Reply:
Yes, but I'd do that in another step to reduce the amount of changes